Vitest error importing `isURL` from `@udecode/plate-serializer-docx` package after bump to 2.4.0

## Description
```
       Error: Cannot find module '/home/rich/source/meetingplan/packages/client/node_modules/validator/lib/isURL' imported from /home/rich/source/meetingplan/packages/client/node_modules/@udecode/plate-serializer-docx/dist/index.mjs
       Did you mean to import validator/lib/isURL.js?
       ⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯
       Serialized Error: { code: 'ERR_MODULE_NOT_FOUND' }
       ⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯⎯[1/5]⎯
```

<!-- A clear and concise description of what the bug is. -->

## Steps to Reproduce
I have a vite based react 18 project with vitest tests.

I made no changes other than to bump `@udecode/plate-core` and `@udecode-plate-serializer-docx` from `24.0.2` to `24.4.0` and my tests under vitest started failing.  Actually running the code, however, seems to work fine.

My use of plate is just to utilize some of the HTML/DOCX related helper functions.

```ts
import {
  cleanDocx,
  isDocxContent,
  isDocxList,
  getDocxListIndent,
  getTextListStyleType,
  getDocxListContentHtml,
} from '@udecode/plate-serializer-docx';
import { preCleanHtml, postCleanHtml } from '@udecode/plate-core';
```

In yarn.log I see 

```
"@udecode/plate-serializer-docx@24.4.0":
  version "24.4.0"
  resolved "https://registry.yarnpkg.com/@udecode/plate-serializer-docx/-/plate-serializer-docx-24.4.0.tgz#ade3932ea13b39f13a16a154434a1ecbe8a2d3e1"
  integrity sha512-OR8HN8dOUrxiaNOtyx6lcU9jby6nWzY4xaiZ6e/BOKySbJsTnkzpS+5PmTFYBornB/phBxxATbsbyFWTBFl+GA==
  dependencies:
    "@udecode/plate-common" "24.4.0"
    "@udecode/plate-heading" "24.4.0"
    "@udecode/plate-indent" "24.4.0"
    "@udecode/plate-indent-list" "24.4.0"
    "@udecode/plate-media" "24.4.0"
    "@udecode/plate-paragraph" "24.4.0"
    "@udecode/plate-table" "24.4.0"
    validator "^13.9.0"

express-validator@^7.0.1:
  version "7.0.1"
  resolved "https://registry.yarnpkg.com/express-validator/-/express-validator-7.0.1.tgz#435fac6b5fa838763f78eca05d2206317b92106e"
  integrity sha512-oB+z9QOzQIE8FnlINqyIFA8eIckahC6qc8KtqLdLJcU3/phVyuhXH3bA4qzcrhme+1RYaCSwrq+TlZ/kAKIARA==
  dependencies:
    lodash "^4.17.21"
    validator "^13.9.0"

validator@^13.11.0, validator@^13.9.0:
  version "13.11.0"
  resolved "https://registry.yarnpkg.com/validator/-/validator-13.11.0.tgz#23ab3fd59290c61248364eabf4067f04955fbb1b"
  integrity sha512-Ii+sehpSfZy+At5nPdnyMhx78fEoPDkR2XW/zimHEL3MyGJQOCQ7WeP20jPYRz7ZCpcKLB21NxuXHF3bxjStBQ==
```

I do also have a dependency on `validator` with version `^13.11.0`, but this is unchanged as works with `@udecode/plate-serializer-docx@24.0.2`.

Rolling back and pinning `@udecode/plate-core` and `@udecode/plate-serializer-docx` to `24.0.2` the tests again succeed.

## Expected Behavior
The code should work as it did before the version bump.

## Environment

- slate: 0.94.1
- slate-react: 0.99.0
- browser: firefox

<!-- POLAR PLEDGE BADGE START -->
## Funding
* You can sponsor this specific effort via a [Polar.sh](https://polar.sh) pledge below
* We receive the pledge once the issue is completed & verified

<a href="https://polar.sh/udecode/plate/issues/2677">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/udecode/plate/issues/2677/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://polar.sh/api/github/udecode/plate/issues/2677/pledge.svg">
</picture>
</a>
<!-- POLAR PLEDGE BADGE END -->

--- a/.changeset/great-files-boil.md
+++ b/.changeset/great-files-boil.md
@@ -1,0 +1,5 @@
+---
+"@udecode/plate-serializer-docx": patch
+---
+
+Fixes #2677: CommonJS validator import in cleanDocXImageElements

--- a/packages/serializer-docx/src/docx-cleaner/utils/cleanDocxImageElements.ts
+++ b/packages/serializer-docx/src/docx-cleaner/utils/cleanDocxImageElements.ts
@@ -1,5 +1,5 @@
 import { hexToBase64, traverseHtmlElements } from '@udecode/plate-common';
-import isURL from 'validator/lib/isURL';
+import validator from 'validator';
 
 import { getRtfImagesMap } from './getRtfImagesMap';
 import { getVShapeSpid } from './getVShapeSpid';
@@ -30,7 +30,7 @@ export const cleanDocxImageElements = (
 
       const alt = element.getAttribute('alt');
 
-      if (typeof alt === 'string' && isURL(alt, { require_protocol: true })) {
+      if (typeof alt === 'string' && validator.isURL(alt, { require_protocol: true })) {
         element.setAttribute('src', alt);
         return true;
       }

